src/js/modules/core/controllers/modal-message/modal-message.tpl.html

```diff
@@ -6,10 +6,10 @@
                     <ion-item class="item-text-wrap item-borderless item-modal-content item-no-bottom-indent">
                         <h2 ng-if="title" class="title" ng-class="titleClass" ng-bind-html="(title | translate)"></h2>
                         <p ng-if="body" class="body" ng-class="bodyClass" browse-to-children>
-                            <span ng-bind-html="(body | translate)"></span>
+                            <span ng-bind-html="(body | translate | nl2br)"></span>
                         </p>
                     </ion-item>
-                    <div class="row">
+                    <div class="row" ng-if="button">
```

Comment:
@romaroid this allows `button: ""` to disable the button, or want to add something like `disableButton: true`?